internal/ansible/controller/reconcile.go

```diff
@@ -63,11 +63,11 @@ type AnsibleOperatorReconciler struct {
 }
 
 // Reconcile - handle the event.
-func (r *AnsibleOperatorReconciler) Reconcile(request reconcile.Request) (reconcile.Result, error) { //nolint:gocyclo
+func (r *AnsibleOperatorReconciler) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) { //nolint:gocyclo
```

Comment:
@asmacdo @camilamacedo86 : Would we like two pass the same context for rest of the functions called inside controller (ie) `mark running`, `mark error` etc ? 
If we cancel the context in reconcile, all the others would get cancelled too. Wasn’t sure if we want that, because the previous code created a new context every time.

Reply:
It shows fine. I cannot see a problem with that. WDYT @fabianvf @asmacdo @estroz ?